frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/breakdown-card/breakdown-body.tsx

```diff
@@ -28,12 +29,29 @@ export const BreakdownCardBody: React.FC<BreakdownBodyProps> = ({
       </EmptyState>
     );
   }
+  if (objectUsed === '0') {
+    return (
+      <EmptyState variant={EmptyStateVariant.full}>
+        <Title className="graph-empty-state__title text-secondary" size="sm">
+          Not enough usage data
+        </Title>
+      </EmptyState>
+    );
+  }
 
-  const available = getCapacityValue(cephUsed, cephTotal, humanize);
-  const usedCapacity = `${humanize(cephUsed).string} used of ${humanize(cephTotal).string}`;
+  const available = getCapacityValue(objectUsed, objectTotal, humanize);
+  const usedCapacity = `${humanize(objectUsed, null, 'GiB').string} used${
+    objectTotal ? ` of ${humanize(objectTotal).string}` : ''
+  }`;
   const availableCapacity = `${available.string} available`;
 
-  const chartData = addAvailable(top5MetricsStats, cephTotal, cephUsed, metricTotal, humanize);
+  const chartData = addAvailable(
+    top5MetricsStats,
+    objectTotal,
+    objectUsed,
+    metricTotal || objectUsed,
```

Comment:
```suggestion
    metricTotal,
```
pass default parameters instead